packages/contracts/contracts/test/testdata/ColosseumTestData.sol

```diff
@@ -254,8 +237,6 @@ library ColosseumTestData {
         pp.pair[1] = 4733049234560598175814075307329400052685708929235536361180057682884349522951;
         pp.pair[2] = 12927143393367435583780384041605583199602086804808450831263369829150907626360;
         pp.pair[3] = 4387398449564854858399845335905273397295727913637629556341886693575315457608;
-        pp.pair[4] = uint256(bytes32(bytes16(_publicInputHash)) >> (8 * 16));
-        pp.pair[5] = uint256(bytes32(bytes16(_publicInputHash << (8 * 16))) >> (8 * 16));
 
```

Comment:
Just quesetion, shouldn't the pair array size be 4?

Reply:
This contains these 4 coordinates. See https://github.com/scroll-tech/halo2-snark-aggregator#21-top-level-sketch for more details.

```math
\displaylines{
[W_{x}.x, W_{x}.y, W_{e}.x, W_{e}.y]
}
```

Reply:
I think he said that the size of `pp.pair` should be 4 since the `pair[4]` and `pair[5]` is removed.
I'll change it. Thx!